lab_1_classify_by_unigrams/main.py

```diff
@@ -80,6 +166,22 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     :return: a dict with a lower-cased loaded profile
     with relative frequencies without unnecessary ngrams
     """
+    if not isinstance(profile, dict) or 'name' not in profile \
+            or 'n_words' not in profile or 'freq' not in profile:
+        return None
+
+    perfect_profile = {'name': profile['name'],
+                       'freq': {}}
+
+    for sequence in profile['freq']:
+        if sequence.lower() in perfect_profile['freq']:
+            perfect_profile['freq'][sequence.lower()] += \
+                profile['freq'][sequence] / profile['n_words'][0]
```

Comment:
`profile['freq'][sequence]` / `profile['n_words'][0]` to variables

Reply:
Done